src/controllers/ChooseAppealReasonController.ts

```diff
@@ -0,0 +1,34 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller } from 'inversify-express-utils';
+import { BaseController } from './BaseController';
+import { FormValidator } from './validators/FormValidator';
+
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { IllnessReasonFeatureMiddleware } from 'app/middleware/IllnessReasonFeatureMiddleware';
+import { schema } from 'app/models/fields/Reason.schema';
+import { CHOOSE_REASON_PAGE_URI } from 'app/utils/Paths';
+import { Navigation } from 'app/utils/navigation/navigation';
+
+const template = 'choose-appeal-reason';
+
+const navigation: Navigation = {
+    previous(): string {
+        return '';
+    },
+    next(): string {
+        return '';
+    },
+    actions: (_: boolean) => {
+        return {
+            continue:'action=continue'
+        };
+    }
+};
+
+@controller(CHOOSE_REASON_PAGE_URI, IllnessReasonFeatureMiddleware, SessionMiddleware, AuthMiddleware)
```

Comment:
Shall we add company auth middleware to not forget? I do not think there is story for that.

Reply:
Adding the CompanyAuth middleware would tie the page into the journey, as attempting to access the page without authenticating first would cause a redirect. To my understanding, for this ticket we should be able to access the page simply by entering the URL. If this isn't the case then I'm happy to make the change.

Reply:
You are right although it is same with being logged in. It does not matter much as long as Richard knows that he needs to authenticate company before entering that screen. If you do not add it make sure it is added somewhere so we do not forget to add it in the future.

Reply:
If I don't add it now, I can add a comment to the Nav ticket to remind whoever picks that up to add it.

Reply:
Okie dokie. I've added a note to the relevant Jira ticket.